compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -372,13 +368,16 @@ chooseMatmulTile(ArrayRef<TileMxNxK> enumeratedTiles, int64_t matmulNarrowM,
   SmallVector<RatedTileMxNxK> ratedTiles;
   ratedTiles.reserve(enumeratedTiles.size());
   int64_t bestPaddingPenalty = INT64_MAX;
+
   int64_t mUB = INT64_MAX;
   int64_t nUB = INT64_MAX;
   int64_t kUB = INT64_MAX;
-  if (!hostDefinedUpperBound.empty()) {
-    mUB = hostDefinedUpperBound[0];
-    nUB = hostDefinedUpperBound[1];
-    kUB = hostDefinedUpperBound[2];
+  if (maxPadding) {
+    mUB =
```

Comment:
can you rename all instances of `UB` (which was confusing because that is often used to mean "undefined behavior") by `padded` and maybe reverse the "word order":
`mUB` --> `paddedM`
etc.

Reply:
done.